test/unit/lodash.spec.js

```diff
@@ -0,0 +1,583 @@
+                }
+            }
+            expect(i).to.eql(1);
+        });
+        it('should not contain own property cube', function() {
+            var obj = params[0];
+            expect(obj).to.have.property('cube');
+            expect(obj.cube).to.be.an('function');
+        });
+        it('should return array property names of functions', function() {
+            expect(_.functionsIn.apply(_, params)).to.eql(['square', 'cube']);
+        });
+    });
+
+    describe('pick', function() {
+        describe('pick1', function() {
+            beforeEach(function() {
+                params = datasets.pick1({
+                    'red': 1,
+                    'green': 2,
+                    'blue': 3,
+                    'yellow': 4
+                });
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0];
+                var element1 = params[1] instanceof Array;
+                expect(element).to.be.an('object');
+                expect(element1).to.eql(true);
+                expect(params).to.have.length(2);
+                for (var i = 0; i < params[1].length; i++) {
+                    expect(params[1][i]).to.be.a('string');
+                }
+            });
+            it('should pick red and yellow apples', function() {
+                expect(_.pick.apply(_, params)).to.eql({
+                    'red': 1,
+                    'yellow': 4
+                });
+            });
+        });
+
+        describe('pick2', function() {
+            beforeEach(function() {
+                params = datasets.pick2({
+                    'red': 1,
+                    'green': 2,
+                    'blue': 3,
+                    'yellow': 4
+                });
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0];
+                var element1 = params[1];
+                expect(element).to.be.an('object');
+                expect(element1).to.be.an('string');
+                expect(params.length).to.eql(2);
+            });
+            it('should pick green apples', function() {
+                expect(_.pick.apply(_, params)).to.eql({
+                    'green': 2,
+                });
+            });
+        });
+    });
+
+    describe('pickBy', function() {
+        describe('pickBy1', function() {
+            beforeEach(function() {
+                params = datasets.pickBy1();
+            });
+            it('should match types of passing elements', function() {
+                var element = params[0] instanceof Object;
+                var element1 = params[1] instanceof Function;
+                expect(element).to.eql(true);
+                expect(element1).to.eql(true);
+                expect(params.length).to.eql(2);
+            });
+            it('should pick ', function() {
+                expect(_.pickBy.apply(_, params)).to.eql({
```

Comment:
This is way too easy. Moreover I can think at least of 3 or more answers possible to met the requirements.